### PR TITLE
{tools}[system/system] gcloud v382.0.0

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ItIsI-Orient** [comment]
Test report by @ItIsI-Orient
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3702.slaking.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/91996187deb291af6a83be873a7b6c58 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/520e9dc946237d452fb73e0730ee6757 for a full test report.

File: easybuild/easyconfigs/g/gcloud/gcloud-382.0.0.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+sources = ["google-cloud-sdk-%(version)s-linux-x86_64.tar.gz"]
+checksums = ['335e5a2b4099505372914acfccbb978cf9d4efd8047bda59f910c26daefd554e']
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+sanity_check_paths = {
+    'files': ["bin/gcloud"],
+    'dirs': []
+}
+
```
Comment (verdurin) [inline easybuild/easyconfigs/g/gcloud/gcloud-382.0.0.eb:21]: @ItIsI-Orient is there a `sanity_check_command` you can add here?

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
centos7.lan - Linux CentOS Linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/6074569a1d8cc0f0fe90bcea9fc58132 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3513.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/fde535ef685b6c20b41b2cdb2263ce8a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**